### ISSUE TITLE
[IE 11 + JAWS 17 and 2018] JAWS announces readonly element as "edit"

### ISSUE THREAD

**Issue by madhaviherle** [FreedomScientific/standards-support#208]
### Summary
We have readonly input field and text area which have the following code.
This is the code: which seems to work well in FF and Chrome browsers with NVDA 18.2.1:
```
<label  id="myTextAreaWithReadOnly-label"  for="myTextAreaWithReadOnly">Read-Only</label>

<p tabindex="0" role="textbox" aria-readonly="true" aria-labelledby="myTextAreaWithReadOnly-label" readonly>This field is read only</p>
```

However, JAWS announces "readonly" sometimes and sometimes says "edit".

### Expected result
I expect JAWS to read this as a readonly field i.e. "<Label> readonly <value>"

### Actual result
JAWS says "<Label> edit <value>" sometimes and sometimes "<Label> readonly <value>"

#### JAWS version and build number
JAWS 17 and 2018

 
#### Operating System and version
Windows

#### Browser and version:
IE 11

**Comment by JAWS-test** [comment]
The problem also occurs with JAWS 2019 and IE 11 (but not with JAWS 2019 and Firefox or Chrome). 

The problem occurs only when JAWS was in Forms mode on the previous item (for example, an input field). 

Also, I noticed that with JAWS 2019 and IE 11, readonly does not work with role=textbox and aria-readonly does not work with input. I.e. ARIA and HTML markup should not be mixed.

**Comment by lamador** [comment]
The issue with the readonly state of the `<p>` element with role="textbox" not being announced was reproducible in the latest version of JAWS (2020.2006.12) and IE11 (11.836.18362.0).
